packages/api-explorer/lib/generateFakeSchema.js

```diff
@@ -8,6 +8,37 @@ jsf.option({
   alwaysFakeOptionals: true,
 })
 
+function generateObjectExample(example, properties) {
+  const generatedExample = { ...properties ? jsf.generate(properties) : {} }
```

Comment:
Why the spread operator?

```suggestion
  const generatedExample = properties ? jsf.generate(properties) : {}
```